src/assets/i18n/fr.json

```diff
@@ -488,24 +488,24 @@
+  "You are trying to open:<br>\n{url}<br><br>\nBecause HTTP to HTTPS redirect is enabled in settings your browser will force HTTPS connection for this URL.<br>\nThis may create issues if app does not support secure connections.<br>\n<br>\nYou can try opening app url in an incognito mode.<br>\nAlternatively you can disable redirect in Settings, clear browser cache and try again.": "Vous essayez d'ouvrir :<br>\n{url}<br><br>\nLa redirection HTTP vers HTTPS étant activée dans les paramètres, votre navigateur forcera la connexion HTTPS pour cette URL.<br>\nCela peut créer des problèmes si l'application ne prend pas en charge les connexions sécurisées.<br>\n<br>\nVous pouvez essayer d'ouvrir l'URL de l'application en mode navigation privée.<br>\nVous pouvez également désactiver la redirection dans les paramètres, vider le cache du navigateur et réessayer.",
+  "You can join the <a href=\"https://www.truenas.com/newsletter/\" target=\"_blank\" class=\"external-link\">TrueNAS Newsletter</a> for monthly updates and latest developments.": "Vous pouvez rejoindre la <a href=\"https://www.truenas.com/newsletter/\" target=\"_blank\" class=\"external-link\">Newsletter TrueNAS</a> pour des mises à jour mensuelles et les derniers développements.",
   "ZFS": "",
-  "ZFS/SED keys synced between KMIP Server and TN database.": "",
+  "ZFS/SED keys synced between KMIP Server and TN database.": "Clés ZFS/SED synchronisées entre le serveur KMIP et la base de données TN.",
   "Zvol": "",
   "by ancestor": "",
-  "expires in {n, plural, one {# day} other {# days} }": "",
-  "iSCSI listen port": "",
-  "iXsystems does not audit or otherwise validate the contents of third-party applications catalogs.     It is incumbent on the user to verify that the new catalog is from a trusted source and that the third-party     properly audits its chart contents. Failure to exercise due diligence may expose the user and their data to     some or all of the following:<br/>    <ul>\n      <li>Malicious software</li>\n      <li>Broken services on TrueNAS host</li>\n      <li>Service disruption on TrueNAS host</li>\n      <li>Broken filesystem permissions on Host or within application</li>\n      <li>Unexpected deletion of user data</li>\n      <li>Unsafe service configuration in application</li>\n      <li>Degradation of TrueNAS host performance and stability</li>\n    </ul>": "",
+  "expires in {n, plural, one {# day} other {# days} }": "Expire dans {n, plural, one {# day} other {# days} }",
```

Comment:
In here you can alter expression inside of {} to match how plural rules work in French.
I.e. could be something like this:
```
Expire dans {n, plural, one {# jour} other {# jours} }
```

You can test here: http://format-message.github.io/icu-message-format-for-translators/editor.html

Reply:
Thanks for the tip and sorry for this mistake.